src/formattedcode/output_spdx.py

```diff
@@ -357,9 +358,11 @@ def write_spdx(
         else:
             file_entry.copyright = SPDXNone()
 
-        package.add_file(file_entry)
+        doc.add_file(file_entry)
+        relationship = Relationship(package.spdx_id + " CONTAINS " + file_entry.spdx_id)
```

Comment:
We still use single quotes for now and what about using interpolation instead?
```suggestion
        relationship = Relationship(f'{package.spdx_id} CONTAINS {file_entry.spdx_id}')
```

Reply:
fixed